superset-frontend/src/explore/components/controls/AdhocFilterControl.jsx

```diff
@@ -275,6 +275,7 @@ export default class AdhocFilterControl extends React.Component {
           clearable
           closeOnSelect
           onChange={this.onChange}
+          onPaste={() => {}}
```

Comment:
Instead of defining this inline, can you define it as a constant NO_OP function above? Better yet, see if we have a NO_OP function exported from a utils file somewhere to use here.

Finally, I think `() => undefined` is prefered to `() => {}` as it doesn't construct an empty object

Reply:
`() => {}` doesn't really return an empty object. It's equivalent to `() => { return }`.

<img src="https://user-images.githubusercontent.com/335541/97356668-d46afc00-1855-11eb-9cd2-2f6f98a55122.png" width="300">

Reply:
I also wouldn't worry about `NO_OP` that much as I'm yet to see it causing any real performance issue.

Reply:
ah yeah, you're totally right. I was mistaken from this eslint rule: https://eslint.org/docs/rules/no-empty-function where they preferred `() => undefined` as it's more obvious that the function is meant to be empty.

That said, it looks like we've defined noOp in a utils file already, so we should probably use that: https://github.com/apache/incubator-superset/blob/ccfd293227105ae1a5a24999a3b75e3db724273b/superset-frontend/src/utils/common.js#L139

Reply:
Done